pom.xml

```diff
@@ -76,21 +76,32 @@
             </Import-Package>
             <Export-Package />
             <_failok>true</_failok>
-            <Embed-Dependency>usercompatibility-jira</Embed-Dependency>
+            <Embed-Dependency>sal-api</Embed-Dependency>
           </instructions>
           <log4jProperties>src/main/resources/log4j.properties</log4jProperties>
         </configuration>
       </plugin>
     </plugins>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
```

Comment:
Why is this duplication needed? <filtering>true</filtering> and <filtering>false</filtering>